content/docs/command-reference/repro.md

```diff
@@ -127,7 +127,7 @@ and only execute the final stage.
 - `-P`, `--all-pipelines` - reproduce all pipelines, for all the stage files
   present in `DVC` repository.
 
-- `--ignore-build-cache` - in cases like `... -> A (changed) -> B -> C` it will
+- `--ignore-run-cache` - in cases like `... -> A (changed) -> B -> C` it will
```

Comment:
do we still have the semantics/description considering that we changed the caching mechanism? (stage vs the whole run) ... is there a summary somwhere btw what exactly do we and how?

Reply:
Yeah, thinking about maybe introducing a separate option for the previous behaviour.

Reply:
Might also use `--no-run-cache`, seems more natural.